src/ui/components.tsx

```diff
@@ -365,6 +365,8 @@ export const HostLabelFilter: React.FC<{
 };
 
 export const OnlyActiveChildren = (props: { children: JSX.Element[]; requestSpec: RequestSpec }): JSX.Element => {
+  type ChildComponent = React.ReactElement<{label: string; onChange(value: unknown): void}, JSXElementConstructor<unknown>>
```

Comment:
Link `onChange` to RequestSpec[T]` where `T` is `RequestSpecKyes` that can be undefined or something along the lines.